src/tilemaker/entrypoint.sh

```diff
@@ -7,6 +7,8 @@ source /opt/pfb/tilemaker/scripts/utils.sh
 
 export TL_MIN_ZOOM="${TL_MIN_ZOOM:-8}"
 export TL_MAX_ZOOM="${TL_MAX_ZOOM:-18}"
+export TL_NUM_PARTS="${TL_NUM_PARTS:-4}"
```

Comment:
Yeah, probably so.  Then we can see what the timings look like for the 17 layer and think about whether it's practical to add another 4x that to do 18.